_unused_topics/upgrade-cluster-version-definition.adoc

```diff
@@ -24,7 +24,6 @@ spec:
   channel: stable-4.3 <1>
   overrides: "" <2>
   clusterID: 0b1cf91f-c3fb-4f9e-aa02-e0d70c71f6e6
-  upstream: https://api.openshift.com/api/upgrades_info/v1/graph
   status: <3>
```

Comment:
nit: this is an indent issue, because `status` is a sibling of `spec`, not a child of `spec`.  But orthogonal to this PR's effort, and this is in `_unused_topics` anyway, so punting for now.